scripts/initial_release_prep.py

```diff
@@ -0,0 +1,312 @@
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script for initial release preparation."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+import os
+import re
+import sys
+
+import python_utils
+import release_constants
+
+from . import common
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_APPENGINE_PATH = os.path.join(
+    _PARENT_DIR, 'oppia_tools', 'google_appengine_1.9.67', 'google_appengine')
+_PYGSHEETS_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'pygsheets-2.0.2')
+_GOOGLE_PATH = os.path.join(_PYGSHEETS_PATH, 'google')
+
+sys.path.insert(0, _APPENGINE_PATH)
+sys.path.insert(0, _PYGSHEETS_PATH)
+
+# pylint: disable=wrong-import-position
+import google # isort:skip
+# pylint: enable=wrong-import-position
+
+# Refer: https://groups.google.com/forum/#!topic/google-appengine/AJJbuQ3FaGI.
+google.__path__.append(_GOOGLE_PATH)
+
+# pylint: disable=wrong-import-position
+import pygsheets # isort:skip
+# pylint: enable=wrong-import-position
+
+RELEASE_CREDENTIALS_FILEPATH = os.path.join(
+    os.getcwd(), 'oppia-release-credentials.json')
+# This file is created when pygsheets asks the user to authenticate.
+AUTH_FILEPATH = os.path.join(os.getcwd(), 'sheets.googleapis.com-python.json')
+
+
+def get_job_details_for_current_release(
+        job_details, job_name_header, month_header, author_email_header,
+        author_name_header, instruction_doc_header):
+    """Returns a list of job names for current release from a list
+    of job details.
+
+    Args:
+        job_details: list(dict). List of dictionary of job details.
+            The keys of each dict are the headers from the jobs
+            spreadsheet.
+        job_name_header: string. The header used for job names.
+        month_header: string. The header used for month of the release.
+        author_email_header: string. The header used for email address of
+            job author.
+        author_name_header: string. The header used for name of
+            job author.
+        instruction_doc_header: string. The header used for instruction
+            doc for running the job.
+
+    Returns:
+        list(dict). The list of job details for current release.
+    """
+    job_details_for_current_release = []
+    # First three letters of the release month. Eg: Nov, Dec etc.
+    current_release_month = datetime.datetime.utcnow().strftime('%h')
+    for job_detail in job_details:
+        if current_release_month in job_detail[month_header]:
```

Comment:
Maybe we'll need to make the month field a dropdown in the form (as not the author is allowed to type in anything I believe). Also, maybe we should do a tolower() before checking?

Reply:
> Maybe we'll need to make the month field a dropdown in the form (as not the author is allowed to type in anything I believe)

Yes, we should do that.

Done.

Reply:
Can you add an AI somewhere (in the release process doc maybe) to fix the form?

Reply:
Done.

Reply:
Done.